event-handler/cli.go

```diff
@@ -62,6 +65,45 @@ type TeleportConfig struct {
 	TeleportKey string `help:"Teleport TLS key file" type:"existingfile" env:"FDFWD_TELEPORT_KEY"`
 }
 
+// Validate performs some basic validation on TeleportConfig that can't be handled using the kong defaulting
```

Comment:
What's "kong defaulting"?

Reply:
Kong supports using struct tags to set default values. Have updated comment to less verbose